internal/controller/chiaseeder/helpers.go

```diff
@@ -7,10 +7,9 @@ package chiaseeder
 import (
 	"context"
 	"fmt"
-	"strconv"
-
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strconv"
```

Comment:
My IDE switched this and resisted putting it back to the old place so this change is unrelated but exists in this PR.